roles/open_zaak_podman/tasks/containers.yml

```diff
@@ -71,6 +72,7 @@
 # Note that there is a bug where stopped pods are destroyed, thereby breaking the
 # infra container ID reference, see:
 # https://github.com/containers/ansible-podman-collections/issues/215
+# update May 11, 2023: This issue might be resolved
```

Comment:
if it's confirmed this is resolved, the comment can be removed completely. If there's still workarounds needed, please include that in the comment.

Reply:
Well the guy who reported the issue never confirmed if it was fixed. I assume the workaround here is using command instead of the containers.podman.podman_pod. I have removed it for now, but will need some further testing to make sure.

Reply:
@sergei-maertens Did some further testing, I'm pretty confident it is no longer required to stop the pod. Restarting the pods through systemd (handler) is also no longer required.

- For clean / new installs systemd will start the containers (with the enable service tasks);
- Systemd file creation works fine with running pods;
- When upping / downing replicas the entire pod is recreated anyway, removing the containers, systemd files are updated to add / remove the replica changes;
- When changing the image tag the container status changes to "stopped", container tag changes and existing systemd will start it again;
- When envars change the containers are recreated already (no changes here on my part) and started again with existing systemd;

So I do not see the need anymore to stop the pod after they are created. Nor do we need to restart the pod with the handler, it just restarts them again when systemd was already handling it.

This would also slightly decrease downtime and remove guaranteed downtime when running the playbook without changes.

Reply:
I'm only reading good news, so let's go for it.